ledger/participant-integration-api/src/main/resources/db/migration/postgres-appendonly/V106__add_rejection_status_proto_column.sql

```diff
@@ -0,0 +1,16 @@
+--  Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+--  SPDX-License-Identifier: Apache-2.0
+
+-- Store the completion status as a serialized Protocol Buffers message of type `google.rpc.Status`.
+
+-- The `rejection_status` contains a Protocol-Buffers-serialized message of type
+-- `google.rpc.Status`, containing the code, message, and further details (decided by the ledger
+-- driver). We do not migrate the `rejection_status_code` and `rejection_status_message` columns,
+-- and so they may contain old data, which means we need to keep them.
```

Comment:
Do I understand correctly?

```suggestion
-- driver). We just rename the `rejection_status_code` and `rejection_status_message` columns,
-- so as to preserve existing data in them.
```